utils/build/docker/ruby/install_ddtrace.sh

```diff
@@ -26,7 +26,7 @@ elif [ $(ls /binaries/ruby-load-from-bundle-add | wc -l) = 0 ]; then
     #
     echo "Install prod version"
     # Support multiple versions of the gem
-    echo "gem 'datadog', '~> 2.0.0.beta2'" >> Gemfile
+    echo "gem 'datadog', '>= 2.0.0.beta2'" >> Gemfile
```

Comment:
This appears to be missing:

```suggestion
    echo "gem 'datadog', '>= 2.0.0.beta2', require: 'datadog/auto_instrument'" >> Gemfile
```

Reply:
Also we might just as well keep `~>`, what's the idea behind changing it to `>=`?

Reply:
Oh, I get it now, `~>` wouldn't allow for e.g `2.1.0`  🤦